app/controllers/questions/personal_info_controller.rb

```diff
@@ -1,6 +1,12 @@
 module Questions
   class PersonalInfoController < QuestionsController
     include AnonymousIntakeConcern
+
+    def edit
+      @show_ssn_field = current_intake&.triage&.id_type_need_help? ? false : true
```

Comment:
Since nil evaluates to false, I feel like you can skip the ternary here and just let this evaluate to the result of `id_type_need_help?`

Reply:
yeah i can't remember why i did this....maybe was feeling paranoid? i would just test it out locally to make sure it works as expected?

Reply:
note that per the PR open for updates to itin in hub, this is not the correct method name for getting to whether they are an itin applicant.